### PR TITLE
Problem: websocket push duplicated messages to client (backport: #370)

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- backport ethermint fix.
- backport websockets integration tests.

wait for the merge of #370 first, and backport the integration tests.

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # Changelog
 
-## UNRELEASED
+*March 2, 2022*
+
+## v0.6.7
```
Comment (yihuang) [inline CHANGELOG.md:5]: release v0.6.7 after this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 ### Bug Fixes
 
 - (rpc) [cronos#352](https://github.com/crypto-org-chain/cronos/pull/352) fix newPendingTransactions subscription deadlock issue, backport the fix in ethermint.
 - (rpc) [cronos#357](https://github.com/crypto-org-chain/cronos/pull/357) fix gasUsed returned by RPC api.
 - (rpc) [cronos#363](https://github.com/crypto-org-chain/cronos/pull/363) fix occasional websocket connection closed due to non-eth tx.
 - (rpc) [cronos#372](https://github.com/crypto-org-chain/cronos/pull/372) Problem: transaction count don't match in different json-rpc apis
+- (rpc) [cronos#]() Fix websocket push duplicated messages to client.
```
Comment (yihuang) [inline CHANGELOG.md:13]:
```suggestion
- (rpc) [cronos#373](https://github.com/crypto-org-chain/cronos/pull/373) Fix websocket push duplicated messages to client.
```

File: go.mod [hunk 3]
```diff
@@ -164,7 +164,7 @@ replace github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethere
 // TODO: remove when ibc-go and ethermint upgrades cosmos-sdk
 replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.7-0.20220214161517-8a26cd10b4be
 
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.2-cronos-11
+replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.7.3-0.20220301141717-2934be7e4cfd
```
Comment (yihuang) [inline go.mod:167]: https://github.com/crypto-org-chain/ethermint/compare/v0.7.2-cronos-11...2934be7e4cfd?expand=1

**Review by JayT106 — COMMENTED** [review]
lgtm

**Review by thomas-nguy — APPROVED** [review]

**Pull request merged**